### PR TITLE
Code review communication server socket

### PR DESCRIPTION
**Pull request by Tymoteuszauke**
- only AppSocketServer (tests)

### CODE REVIEW

File: server/src/main/java/com/blackship/blacksheep/communication/network/AppServerSocket.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package com.blackship.blacksheep.communication.network;
+
+import org.slf4j.Logger;
```
Comment (arturAR) [inline server/src/main/java/com/blackship/blacksheep/communication/network/AppServerSocket.java:3]: 👍

**Review by arturAR — APPROVED** [review]
Clean and clear code! 👍

File: server/src/main/java/com/blackship/blacksheep/communication/network/AppServerSocket.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import org.slf4j.LoggerFactory;
```
Comment (piotr921) [inline server/src/main/java/com/blackship/blacksheep/communication/network/AppServerSocket.java:4]: Not used imports 😞

File: server/src/test/java/com/blackship/blacksheep/communication/network/AppServerSocketTest.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+import static org.mockito.Mockito.mock;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
```
Comment (piotr921) [inline server/src/test/java/com/blackship/blacksheep/communication/network/AppServerSocketTest.java:13]: Unused imports

**Review by piotr921 — APPROVED** [review]

**Pull request closed without merging**